### ISSUE TITLE
Palette 2 Integration

### ISSUE THREAD

**Issue by cfelicio** [cadriel/fluidd#249]
This is very fresh off the press, but klipper now officially supports the Palette 2 via connected mode: 

https://github.com/KevinOConnor/klipper/pull/4057

Before this feature, you would need to use Octoprint plugins to talk to the Palette, so it was pretty much mandatory to use Octoprint if you had a Palette.

The feature works really well, and I'm wondering if it's possible at some point to have the info the Palette generates on the Fluidd interface? Similar to what is currently displayed on the Octoprint plugin:

![image](https://user-images.githubusercontent.com/7227683/114976432-d09a5880-9e3a-11eb-87dd-686132b6a1b0.png)

Appreciate your input!

**Comment by cadriel** [comment]
Yep, I think this is valid. Its on the roadmap, no timeframe as yet.

**Comment by chippawah** [comment]
@cfelicio Are you currently able to use the palette with Fluidd and it just doesn't show the output from the palette? I'm looking to replace Octoprint with Fluidd when I setup my Palette but wonder if I should stick with Octo until this issue is resolved.

**Comment by cfelicio** [comment]
@chippawah Palette works fine with Fluidd as long as you configure it on the klipper.cfg. I can still see the pings on the Palette itself, so this is just a "nice to have".

**Comment by ETE-Design** [comment]
Will this ever be implemented?

**Comment by pedrolamas** [comment]
The reality is that this is not a core priority for Fluidd.

I personally don't have the hardware to test this so that makes things even harder, but if someone on the community wants to provide a code contribution for this (and ensure that either they or we can maintain this feature) we might take a look and see how we can integrate it.
